docker-compose.yml

```diff
@@ -4,15 +4,14 @@ services:
 
   hf:
     build: .
-    ports:
-      - "4567:4567"
```

Comment:
Do we know of any external dependencies on these ports for this service?

Reply:
I think this was copy-pasta - it doesn't even start a service that runs on those ports.